spec/unit/primitive_consul_maintenance_spec.rb

```diff
@@ -41,7 +41,7 @@
     let(:service_id) { 'service_test' }
     let(:choregraphie) do
       Choregraphie::Choregraphie.new('test') do
-        consul_maintenance service_id: service_id, reason: 'Testing', consul_token: 'foo', check_interval: 0
+        consul_maintenance service_id:, reason: 'Testing', consul_token: 'foo', check_interval: 0
```

Comment:
Not commenting on all occurances of this, but if you add the "EnforcedShorthandSyntax: never" cop config, please remove these shorthand syntax ("service_id:,"

Reply:
Done